src/style.css

```diff
@@ -1,3 +1,83 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  display: flex;
+}
+
+.header {
+  display: flex;
+  box-sizing: border-box;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  padding: 0 50px;
+  width: 100%;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-list-link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  gap: 20px;
+}
+
+.nav_link:hover,
+.nav_link.is-active {
+  color: var(--link-color-hover);
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.nav-item {
+  margin-right: 20px;
+  display: inline-block;
+}
+
+.nav-item:last-child {
+  margin: 0;
+}
+
+.logo-img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.is-active {
+  position: relative;
+}
+
+.is-active::after {
+  display: block;
+  content: '';
+  width: 100%;
+  height: 4px;
+  bottom: 0;
+  border-radius: 8px;
+  position: absolute;
+  background-color: #00acdc;
```

Comment:
Create a variable for repeated color

Reply:
Still not fixed 
<img width="869" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/f88be8ef-aa19-4d9b-95cb-8631ff84cfae">